blog/2023-09-19-nushell_0_85_0.md

```diff
@@ -0,0 +1,528 @@
+to make the command less easy to understand in
+[#10258](https://github.com/nushell/nushell/pull/10258)
+[#10259](https://github.com/nushell/nushell/pull/10259) and
+[#10270](https://github.com/nushell/nushell/pull/10270).
+
+Thanks to him again, one can now quit the `explore`r by hitting any of `ctrl+c`, `ctrl+d` or
+`ctrl+q` (see [#10257](https://github.com/nushell/nushell/pull/10257)).
+
+## Making Nushell easier and better to live in
+### Conditional sourcing and much more
+The Nushell core team has been asked quite a lot of times to be able to do parse-time conditional
+`source`ing` or `use`ing in the configuration, e.g. to load different things depending on the
+system:
+```nushell
+if $nu.os-info.name == "windows" {
+    source "my_windows_config.nu"
+} else {
+    source "my_unix_config.nu"
+}
+```
+
+This was not possible until the excellent work proposed by [@kubouch](https://github.com/kubouch):
+- in [#9499](https://github.com/nushell/nushell/pull/9499), he made the evaluation of many Nushell
+    structures possible at parse-time, given that arguments and commands are all constants
+- with [#10326](https://github.com/nushell/nushell/pull/10326), `if` is now a valid parse-time
+    constant structure
+- finally, [#10160](https://github.com/nushell/nushell/pull/10160) makes the `$nu` built-in variable
+    a true constant :partying_face:
+
+What does this all mean? That one can now write something very similar to the proposed snippet
+above!
+```nushell
+const WINDOWS_CONFIG = "my_windows_config.nu"
+const UNIX_CONFIG = "my_unix_config.nu"
+
+const ACTUAL_CONFIG = if $nu.os-info.name == "windows" {
+    $WINDOWS_CONFIG
+} else {
+    $UNIX_CONFIG
+}
+
+source $ACTUAL_CONFIG
+```
+
+### Improving accessibility
+Up until now, an issue for users using a screen reader was that Nushell errors and tables are quite
+fancy with some unicode characters.
+The issue is that screen reader might have trouble reading them, making the understanding of what's
+going on the REPL very hard...
+
+For tables, it's quite easy, you can set `$env.config.table.mode` to something like `"basic"` or
+`"none"` and you should be good!
+But errors remained fancy
+```nushell
+Error: nu::shell::external_command
+
+  × External command failed
+   ╭─[entry #4:1:1]
+ 1 │ foo
+   · ─┬─
+   ·  ╰── did you mean 'for'?
+   ╰────
+  help: No such file or directory (os error 2)
+```
+
+[@JoaquinTrinanes](https://github.com/JoaquinTrinanes) did implement screen reader-friendly errors
+in [#10122](https://github.com/nushell/nushell/pull/10122) which will hopefully make the experience
+a lot better when it comes to errors.
+
+## More support for more platforms
+In this release, [@dead10ck](https://github.com/dead10ck) made it possible to use Nushell in Termux
+([#10013](https://github.com/nushell/nushell/pull/10013)) and fixed a bug on Android
+([#10225](https://github.com/nushell/nushell/pull/10225)).
+
```

Comment:
```suggestion
## Improved history isolation
Like many other shells, history between open sessions can be isolated in nushell.
However, the former implementation had the disadvantage that it also blocked access to history of former sessions.
It also didn't isolate hints.
This is now fixed with [!10402](https://github.com/nushell/nushell/pull/10402) by [@Hofer-Julian](https://github.com/Hofer-Julian).
Since this was merged shortly before the release, this feature isn't the default yet.
The `file_format` has to be changed to "sqlite" and `isolation` has to be set to `true`: https://github.com/nushell/nushell/blob/39d93b536a02c95c263925a47b9e4c34587ce021/crates/nu-utils/src/sample_config/default_config.nu#L202.
Please test this so we can fix problems before history isolation becomes the default.

```